internal/postprocessor/main.go

```diff
@@ -686,23 +684,22 @@ func extractHashFromLine(line string) string {
 }
 
 func updateCommitTitle(title, titlePkg string) string {
-	var newTitle string
 	var breakChangeIndicator string
+	titleParts := strings.Split(title, ":")
+	commitPrefix := titleParts[0]
+	msg := strings.TrimSpace(titleParts[1])
 
-	titleSlice := strings.Split(title, ":")
-	firstTitlePart := titleSlice[0]
-	secondTitlePart := strings.TrimSpace(titleSlice[1])
-
-	if strings.HasSuffix(firstTitlePart, "!") {
+	// If a scope is already provided, remove it.
```

Comment:
Is there a test for this? I looked through the `.input` files and didn't see a scope, but I may have missed it.

Reply:
The last test-case is net-new and catches this case. Because the scope was being duplicated in PRs 😄

Reply:
Looks good!